miner/worker.go

```diff
@@ -1145,6 +1145,17 @@ loop:
 			// However, after `ErrUnknown`, ccc might remain in an
 			// inconsistent state, so we cannot pack more transactions.
 			circuitCapacityReached = true
+			txNumInCcc, err := w.circuitCapacityChecker.GetTxNum()
```

Comment:
Is `txNumInCcc` the count before this last (failing) transaction? Or does it include this transaction?

Reply:
I checked [this row_usages.push code of scroll-prover](https://github.com/scroll-tech/scroll-prover/blob/main/prover/src/zkevm/capacity_checker.rs#L231) as @lispc mentioned, it seems to be the count before last failed.

@lispc deleted `w.current.tcount + 1` in commit [099504c](https://github.com/scroll-tech/go-ethereum/pull/505/commits/099504c9c22152414b9cea5a166d83f4203fb6ea), please correct me if I was wrong.